src/movies.js

```diff
@@ -1,16 +1,150 @@
 // Iteration 1: All directors? - Get the array of all directors.
 // _Bonus_: It seems some of the directors had directed multiple movies so they will pop up multiple times in the array of directors. How could you "clean" a bit this array and make it unified (without duplicates)?
 
+
+
+function getAllDirectors(movies){
+    return movies.map((movie) => movie.director).filter((value, index, self) => self.indexOf(value) === index);
```

Comment:
Very nice, all in one line, however the filter in this case is not trying to solve any Jasmine Test. Removing that part will throw the same result.